gensim/models/phrases.py

```diff
@@ -848,7 +848,10 @@ def score_item(self, worda, wordb, components, scorer):
 
         """
         try:
-            return self.phrasegrams[tuple(components)][1]
+            if list(self.phrasegrams.values())[0].__class__ is tuple:
```

Comment:
1. this check is very tricky to just check that value is instance of tuple. Python has **isinstance** builtin function for that.
2. creation of list of values is not easy operation. It would be better to avoid this.

```suggestion
            score = self.phrasegrams[tuple(components)]
            return score[-1] if isinstance(score, tuple) else score
```

Reply:
But *isinstance* has to be applied to dictionary value, which requires using *list*.